pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,7 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            cmd += (" \"" + PF.FileUtils.sanitize_path (s) + "\"");
```

Comment:
Bearing mind that the malfactor does not have physical access and the uri is that of a file that has been downloaded with e.g. FireFox I am not sure how much of a risk this is.  If specific examples of dangerous "uris" can be given then maybe the sanitise function can be strengthened.